src/app/providers/fault-summary/fault-summary.ts

```diff
@@ -148,9 +148,13 @@ export class FaultSummaryProvider {
         return FaultSummaryCatBEHelper.getSeriousFaultsCatBE(data);
```

Comment:
Can we create a FaultSummaryCatManoeuvreHelper class like is done for other categories, the code inside the C helper is largely redundant for manoeuvres so want to make sure we have separation of responsibility. 

Additionally, can we also add in the D manoeuvre categories to the switch as it will save work in future